src/controllers/gameController.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+
+export const gameController = {
+  patchingMiddleware(
+    req: express.Request,
+    res: express.Response,
+    next: express.NextFunction
+  ) {
+    if (req.originalUrl === '/games/pacman/file/to/patch') {
```

Comment:
If you end up with several files, you might end up wanting to check to see if a patched file exists in a subdir, and otherwise serve the original.

(no direct action needed)